orb-slot-ctrl/tests/orb_slot_ctrl.rs

```diff
@@ -0,0 +1,77 @@
+use orb_slot_ctrl::test_utils::Fixture;
+use orb_slot_ctrl::{RootFsStatus, Slot};
+
+#[test]
+fn it_gets_current_slot() {
+    let fx = Fixture::new();
+    let slot = fx.slot_ctrl.get_current_slot().unwrap();
+    assert_eq!(slot, Slot::A)
+}
+
+#[test]
+fn it_gets_inactive_slot() {
```

Comment:
Question: Do you have an opinion on whether test functions should start their names with `test_` or not? In some cases I think it could disambiguate cases where we forgot to annotate with #[test]. But IDK.

Reply:
Nah, I don't think its necessary... starting tests with `it_` is a convention in some places as well, but I don't really think it matters since we have to annotate with `#[test]`.